ci/get-unifi.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+# Copyright (c) 2025 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+set -euo pipefail
+
+OS_TYPE="$(uname -s | tr A-Z a-z)"
+if [[ $(uname -m) == 'x86_64' ]]; then
+  CPU_ARCH='amd64'
+else
+  CPU_ARCH="$(uname -m)"
+fi
+[[ ! -d "${HOME}/bin" ]] && mkdir -pv "${HOME}/bin"
```

Comment:
What-s the point of creating this directory, when we store in `${HOME}/.unifi/bin` instead?